packages/terra-pills/src/private/_RollupPill.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {
+  useRef,
+} from 'react';
+import {
+  KEY_SPACE, KEY_RETURN,
+} from 'keycode-js';
+import { injectIntl } from 'react-intl';
+import classNamesBind from 'classnames/bind';
+import PropTypes from 'prop-types';
+import ThemeContext from 'terra-theme-context';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+import styles from './Pill.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * Determines if the Pill List is rolled up or not.
```

Comment:
Lines 18, 20 and 28: is rolling up and collapsing the same concept or different concepts? If they're different is this difference doc'ed anywhere? It's not obvious to me what the difference would be.

Reply:
They are same concepts. Generally the pills are collapsed and it is indicated by a rollUp pill Ex:(6 more...).